src/test/kotlin/lotto/domain/LottoNumbersTest.kt

```diff
@@ -2,18 +2,18 @@ package lotto.domain
 
 import io.kotest.core.spec.style.StringSpec
 import io.kotest.matchers.shouldBe
-import lotto.domain.LottoNumbers.Companion.LOTTO_NUMBER_COUNT
+import lotto.domain.Lotto.Companion.LOTTO_NUMBER_COUNT
 
 class LottoNumbersTest : StringSpec({
```

Comment:
```suggestion
class LottoTest : StringSpec({
```
테스트 클래스 이름도 챙겨주시면 좋습니다. 😄